python/src/aiconfig/default_parsers/dalle.py

```diff
@@ -124,7 +138,7 @@ async def serialize(
                 model=model_metadata, parameters=parameters, **kwargs
             ),
         )
-        return [prompt]
+        return prompt
```

Comment:
Hmmmm, I think we should return an array always from `serialize`

Reply:
Thanks for clarifying, did diff here to fix issue: https://github.com/lastmile-ai/aiconfig/pull/632